jac-uk/apply
# JAC-KIT 107 Trim whitespace from TextField component

## What's included?
On the platform candidates are adding leading spaces to text boxes so they appear at the top of lists of candidates in exercises and other areas in the platform e.g Merit Lists, Sift Packs etc.

When candidates enter data on the apply side leading spaces need to be removed before saved and or ignore leading spaces in sorting order.

The fix trims all leading and trailing spaces for the TextField component.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Try adding leading/trailing whitespace to text inputs and ensure that the ensuing saved data does not have leading/trailing whitespace.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
